examples/wandb_sweeps/train_wandb_sweep.py

```diff
@@ -42,7 +42,10 @@ def train_sweep(args: argparse.Namespace) -> None:
     best_checkpoint = train.train(trainer, model, datamodule, args.train_from)
     util.log_info(f"Best checkpoint: {best_checkpoint}")
 
-
+   # cleanup
```

Comment:
Please remove the blank line above, and instead make sure there are two blank lines between this and the following function (e.g., around line 48). 

The comment style should be something like: `# Cleans up.`: a sort of declarative sentence, punctuated as such. But that's not very descriptive. I would suggest something like the following:

```python
# Explicitly deallocates model and clears the cache, based on the following suggestion:
# 
#     https://github.com/wandb/wandb/issues/1247#issuecomment-1457737657
```

When you do something based on a "tip" it's good style to link to said tip, to politely give credit but also to provide context.

Reply:
Yes, it helped with the memory leaks without breaking anything.